openmc/deplete/results.py

```diff
@@ -164,6 +165,68 @@ def get_atoms(
 
         return times, concentrations
 
+    def get_mass(self,
+        mat: typing.Union[Material, str],
+        nuc: str,
+        nuc_units: str = "g",
+        time_units: str = "s"
+    ) -> Tuple[np.ndarray, np.ndarray]:
+        """Get mass of nuclides over time from a single material
+
+        .. versionadded:: 0.13.4
+
+        .. note::
+            Initial values for some nuclides that do not appear in
+            initial concentrations may be non-zero, depending on the
+            value of the :attr:`openmc.deplete.CoupledOperator.dilute_initial`
+            attribute. The :class:`openmc.deplete.CoupledOperator` class adds
+            nuclides according to this setting, which can be set to zero.
+
+        Parameters
+        ----------
+        mat : openmc.Material, str
+            Material object or material id to evaluate
+        nuc : str
+            Nuclide name to evaluate
+        nuc_units : {"g", "g/cm3", "kg"}, optional
+            Units for the returned concentration. Default is ``"g"``
+
```

Comment:
No need to list default since it is in the method signature
```suggestion
        mass_units : {"g", "g/cm3", "kg"}, optional
            Units for the returned mass.
```